### DISCUSSION TITLE
老百姓的诉求而已，不要过分解读

### DISCUSSION THREAD

**Discussion by SeeLuminita** [WeNeedHome/SummaryOfLoanSuspension#695] [General]
业主没有拿到房子，还要还贷款，怎么看都是受害者，没必要把问题上升到政治领域。

**Comment by yalo-ddaa** [comment]
然而老爷觉得你们组织活动就算是政治行为了。
别自欺欺人了，就算你们对着白衣人大喊”我爱党爱国“，他们照样打你。因为你侵犯了老爷的利益。

**Comment by ghost** [comment]
没事，你不想搞政治，支共就会搞你。

**Comment by pengyidong** [comment]
都出动警察了，为啥不能说是政治事件

**Reply by yanwuguangjun** [reply]
什么时候出动警察了   瞎带节奏 别乱造谣

**Comment by clleo888** [comment]
业主没有拿到房子，还要还贷款，怎么看都是受害者，这已经是政治了, 不要把其他人当傻子!
被割而不作声,是生活, 
被割而出声, 是政治! 老共就是这样无理!

**Comment by MYWmiss** [comment]
断供就是反国家反政府行为啊，你做着这么肮脏龌龊的事，还说自己是普通诉求？

**Comment by miaodi** [comment]
没有过分解读，我只是来看笑话的。被铁拳锤死活该，你不关心政治，等维尼把桂枝搞烂了我回去捡便宜，不知道有多开心。
